python/pyfeelpp/feelpp/alg.cpp

```diff
@@ -276,7 +276,7 @@ PYBIND11_MODULE(_alg, m )
         .def( "addVector", static_cast<void ( Vector<double>::* )
         ( std::shared_ptr<Vector<double>> const& , std::shared_ptr<MatrixSparse<double>> const&  )
         >( &Vector<double>::addVector ), "add the product of a MatrixSparse and a vector to this vector" )
-        .def( "add", [](Vector<double> &v, double a, Vector<double> const& w){ v.add(a, w); }, "add a scalar*vector to this vector" )
+        .def( "addVector", [](Vector<double> &v, double a, Vector<double> const& w){ v.add(a, w); }, "add a scalar*vector to this vector" )
```

Comment:
@prudhomm With the wrapping `add(List[float] alphas, List[vector] vs)` computing $u += \sum_i \alpha_i vs[i]$, the initial `add(float alpha, vector v)` computing $u += \alpha u$ was no longer included in the wrapped class.
A workaround I found is to rename this method to `addVector` so it can be wrapped correctly. Maybe there is another way to do it...

#2139